EligibilityCheckOrchestrator/index.ts

```diff
@@ -59,6 +64,7 @@ export const getMessageType = (
     : none;
 };
 
+// tslint:disable-next-line: no-big-function
```

Comment:
temporary disabled this rule cause yield operator limits fp-ts usage for code compression